packages.el

```diff
@@ -81,6 +81,13 @@ Each entry is either:
                (configuration-layer/package-used-p 'lsp-mode))
           (require 'imandra-mode-lsp)))))
 
+(defun imandra/init-ocamlformat ()
```

Comment:
Do you need to add `ocamlformat` to the `imandra-packages` list above?

Should this be `imandra/post-init-ocamlformat`, as `ocamlformat` should be owned by the `ocaml` layer and not the `imandra` layer?

Reply:
Yeah, I think `ocamlformat` probably should be added to the packages, looking at the `require`s form the `imandra-mode-ocamlformat.el`: https://github.com/imandra-ai/imandra-mode/blob/master/imandra-mode-ocamlformat.

And I was copying how the `ocaml` layer adds the hook: https://github.com/syl20bnr/spacemacs/blob/2dadee57110fc9db43756fcad08af9ac3ae7c9fa/layers/%2Blang/ocaml/packages.el#L151-L156, but I think that the `imandra` layer _does_ own `ocamlformat`, since it's `require`d by `imandra-mode-ocamlformat`.

Reply:
👍 I thought that only one layer is allowed to define `<layer>/init-<package>` but maybe that's not the case any more, or it never was.